tests/unit/Dns/Cname/Requests/UpdateRequestTest.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+namespace Tests\Unit\Dns\Cname\Requests;
+
+use PHPUnit\Framework\TestCase;
+use ResellerClub\Dns\Cname\Requests\UpdateRequest;
+use ResellerClub\TimeToLive;
+
+class UpdateRequestTest extends TestCase
+{
+    public function testInstantiation()
+    {
+        $ttl = new TimeToLive(TimeToLive::MINIMUM_TTL);
+        $request = new UpdateRequest(
+            'test.com',
+            'www',
+            'cname.old.com',
+            'cname.new.com',
+            $ttl
+        );
+
+        $this->assertInstanceOf(UpdateRequest::class, $request);
+    }
+
+    public function testInstantiationWithoutTTL()
+    {
+        $request = new UpdateRequest(
+            'test.com',
+            'www',
+            'cname.old.com',
+            'cname.new.com'
+        );
+
+        $this->assertInstanceOf(UpdateRequest::class, $request);
+    }
+
+    public function testGetters()
+    {
+        $ttl = new TimeToLive(TimeToLive::MINIMUM_TTL);
```

Comment:
If this is going to be a common way to create the object then please create a named constructor i.e. 
```
$ttl = TimeToLive::minimumTtl();
```
```
public static function minimumTtl(): self
{
    return new self(self::MINIMUM_TTL);
}
```

Reply:
Resolved in latest commit.